pkg/config/struct_fields.go

```diff
@@ -0,0 +1,92 @@
+package config
+
+import (
+	"reflect"
+	"strings"
+
+	"github.com/treeverse/lakefs/pkg/logging"
+)
+
+const (
+	FieldMaskedValue   = "******"
+	FieldMaskedNoValue = "------"
+)
+
+// MapLoggingFields returns all logging.Fields withe formatted based on our configuration struct 'dot.name.key' with
+// the associated value. Support squash and secret to skip print out of secrets.
```

Comment:
This comment seems to have collected some typos.  Maybe something like
```suggestion
// MapLoggingFields returns all logging.Fields formatted based on our configuration keys 'dot.name.key' with
// associated values. Supports squash, and secret to skip printing out secrets.
```